### DISCUSSION TITLE
Showing Even Labels

### DISCUSSION THREAD

**Discussion by Abhi-ak** [simeydotme/svelte-range-slider-pips#66] [Formatting / Styling] [answered]
Hi, I am using this module in react project. The requirement is to show even years with label [2006,2008,2010.....2020] and odd years with just pips and no label. What is the prop/style config for this?

**Comment by Abhi-ak** [comment]
![image](https://user-images.githubusercontent.com/38856345/119022979-d990c380-b9be-11eb-9b63-c078c474abdb.png)

Here I need to hide 2007, 2009, 2011...2019 lables but need to show pip

**Comment by Abhi-ak** [comment]
@simeydotme

**Comment by simeydotme** [comment]
## Admonishing;
@Abhi-ak  please read a github repo [Contributing.md](https://github.com/simeydotme/svelte-range-slider-pips/blob/main/CONTRIBUTING.md) before posting questions :) 

## Answer; 
Use CSS.
https://svelte.dev/repl/f7c2141a4d924ebc9aa03ae0fbe28c4e?version=3.32.1

**Comment by Abhi-ak** [comment]
Ya, I got it.. Thanks

**Comment by simeydotme** [comment, marked as answer]
This is demo'd at; https://simeydotme.github.io/svelte-range-slider-pips/en/examples/pip-labels#only-nth-labels
